.pre-commit-config.yaml

```diff
@@ -12,21 +12,20 @@ repos:
       - id: trailing-whitespace
       - id: check-ast
       - id: check-json
+      - id: check-yaml
       - id: check-merge-conflict
-      - id: no-commit-to-branch
-        args: [ --branch, master ]
   - repo: https://gitlab.com/pycqa/flake8
     rev: 3.8.4
     hooks:
       - id: flake8
         additional_dependencies: [
-            'flake8-bugbear==20.11.1',
```

Comment:
removed pinned - i assume the latest version works for us

Reply:
Yep agree